benchcab/job_script.py

```diff
@@ -30,30 +33,26 @@ def create_job_script(
         print("Current directory structure unknown on Gadi")
         sys.exit(1)
 
+    print(f"Creating PBS job script to run FLUXNET tasks on compute nodes: {QSUB_FNAME}")
     with open(QSUB_FNAME, "w", encoding="utf-8") as file:
-        file.write("#!/bin/bash\n")
-        file.write("\n")
-        file.write("#PBS -l wd\n")
-        file.write(f"#PBS -l ncpus={NCPUS}\n")
-        file.write(f"#PBS -l mem={MEM}\n")
-        file.write(f"#PBS -l walltime={WALL_TIME}\n")
-        file.write("#PBS -q normal\n")
-        file.write(f"#PBS -P {project}\n")
-        file.write("#PBS -j oe\n")
-        file.write(f"#PBS -M {email_address}\n")
-        file.write("#PBS -l storage=gdata/ks32+gdata/wd9+gdata/hh5"
-                   f"+gdata/{project}+{curdir_root}/{curdir_proj}\n")
-        file.write("\n")
-        file.write("\n")
-        file.write("\n")
-        file.write("\n")
-        file.write("module purge\n")
-        file.write("module use /g/data/hh5/public/modules\n")
-        file.write("module load conda/analysis3-unstable\n")
-        for module_name in modules:
-            file.write(f"module add {module_name}\n")
-        file.write(f"benchcab fluxnet-run-tasks --no-submit --config={config_path}")
-        file.write("\n")
+        file.write(f"""#!/bin/bash
+#PBS -l wd
+#PBS -l ncpus={NCPUS}
+#PBS -l mem={MEM}
+#PBS -l walltime={WALL_TIME}
+#PBS -q normal
+#PBS -P {project}
+#PBS -j oe
+#PBS -M {email_address}
+#PBS -l storage=gdata/ks32+gdata/wd9+gdata/hh5+gdata/{project}+{curdir_root}/{curdir_proj}
```

Comment:
```suggestion
#PBS -l storage=gdata/ks32+gdata/hh5+gdata/{project}+{curdir_root}/{curdir_proj}
```
Remove `wd9` now if we change the formatting here